test/WebSites/ApplicationModelWebSite/Controllers/HomeController.cs

```diff
@@ -17,5 +20,22 @@ public object GetHelloWorld([FromHeader] string helloWorld)
         {
             return ControllerContext.ActionDescriptor.Properties["source"].ToString() + " - " + helloWorld;
         }
+
+        [HttpGet("Home/CannotBeRouted")]
+        [HttpGet("Home/CanBeRouted")]
+        [SuppressLinkGenerationConvention]
+        public object SuppressLinkGeneration()
```

Comment:
The only way to configure this is via conventions \ ApplicationModel, and I'd rather keep the tests for inbound and outbound together.